File: C/functioneg/Prime.c
/*
    Author : Riya Patel
    DOC : 18 july,2023
    Objective : find if the number is prime or not
*/

#include <stdio.h>
int prime(int num)
{ 
    int flag = 0;
    for (int j = 2; j < num; j++)
    {
        if (num % j == 0)
        {
            flag = 1;
            break;
        }
    }
    if (flag == 0 && num > 1)
    {
        printf("\n%d num is prime", num);
    }
    else
    {
        printf("\n%d num is not prime", num);
    }
   
}

int main()
{
    int num;
    printf("Enter a number: ");
    scanf("%d", &num);
    prime(num);
    return 0;
}

File: python/python1/loop/prime.py
# num1=int(input("Enter first number: "))
# num2=int(input("Enter last number: "))
# flag=0
# for i in range(num1,num2):
#     j=1
#     for j in range(2,i):
#         if(i%j==0):
#             flag=1
#             break
#         else:
#             flag=0

#     if(flag==0 and i>1):
#         print(i," is prime")
#     else:
#         continue

num=int(input("Enter first number: "))
flag=0
if num==1:
    flag=1
for i in range(2, num):
    if(num%i == 0):
        flag = 1
        break
    else:
        flag = 0

if (flag == 0):
    print(num, " is prime")
else:
    print(num, " is not prime")
